### PR TITLE
Adjust Replication fetching

### PR DESCRIPTION
**Pull request by kongzii**

### CODE REVIEW

File: prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py [hunk 1]
```diff
@@ -66,7 +65,11 @@ def omen_replicate_from_tx(
         10 if market_type == MarketType.POLYMARKET else 1000,
         market_type,
         filter_by=FilterBy.OPEN,
-        sort_by=sort_by,
+        sort_by=(
+            SortBy.NONE
+            if market_type == MarketType.POLYMARKET
```
Comment (kongzii) [inline prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py:70]: Polymarket doesn't allow sorting by closing soonest

File: prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py [hunk 2]
```diff
@@ -66,7 +65,11 @@ def omen_replicate_from_tx(
...
+            else SortBy.CLOSING_SOONEST
```
Comment (kongzii) [inline prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py:71]: I noticed today that on Manifold, when replication form markets closing in less than a week, we have 6 markets available with default sorting or 460 markets with this sorting. So this gives more opportunities to actually go through our filters (is_predictable, is_predictable_without_description) and replicate something.

File: prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py (hunk 1 above)
Comment (gabrielfior) [reply prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py:70]: I would suggest sorting by Volume - this way, we have the "most interesting" (or relevant) markets.

Maybe for another ticket - when creating the market, can we create it with different odds than the default 50/50? I would say we should copy the odds from the source market.

Comment (kongzii) [reply prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py:70]: It's further sorted by volume just below this: 

Screenshot by Dropbox Capture

> Maybe for another ticket - when creating the market, can we create it with different odds than the default 50/50? I would say we should copy the odds from the source market.

Yes that's possible

Comment (gabrielfior) [reply prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py:70]: @coderabbit create a new issue out of this discussion

Comment (gabrielfior) [reply prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py:70]: Referenced question 2 [here](https://github.com/gnosis/prediction-market-agent-tooling/issues/406). Closing this.

**Review by gabrielfior — APPROVED** [review]

**Pull request merged**